admin/tools/docker/integration-tests/localhost/pull.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Download Docker container related to Qserv integration tests
+
+# @author  Fabrice Jammes, IN2P3/SLAC
+
+. ./env.sh
```

Comment:
Is the intent that `env.example.sh` is renamed by the user to `env.sh`?  If so, a comment to that affect might be useful.